pkg/utils/metadata_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	testingMetadataClient "github.com/google/knative-gcp/pkg/gclient/metadata/testing"
+)
+
+func TestProjectID(t *testing.T) {
```

Comment:
These tests could also benefit from a way to mock errors from the metadata client. Currently there's no coverage of the error cases.